ui/src/app/settings/containers/Users/Users.js

```diff
@@ -1,13 +1,23 @@
-import { connect } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { useEffect } from "react";
 
+import actions from "app/settings/actions";
+import selectors from "app/settings/selectors";
 import { UserShape } from "app/base/proptypes";
 import baseSelectors from "app/base/selectors";
 import MainTable from "app/base/components/MainTable";
 
-export const Users = ({ authUser, users }) => {
+const Users = () => {
+  const dispatch = useDispatch();
+  const users = useSelector(selectors.users.get);
+  const authUser = useSelector(baseSelectors.auth.getAuthUser);
```

Comment:
The new redux hooks API obviates the `connect` HoC, but introduces some coupling.